src/bootstrap/src/core/sanity.rs

```diff
@@ -39,6 +39,7 @@ pub struct Finder {
 #[cfg(not(feature = "bootstrap-self-test"))]
 const STAGE0_MISSING_TARGETS: &[&str] = &[
     // just a dummy comment so the list doesn't get onelined
+    "armv7-rtems-eabihf",
```

Comment:
@onur-ozkan do these get removed as part of the stage0 bump? Or should they get a `// #[cfg(bootstrap)]` to make sure they get caught?

Reply:
Based on the errors, seems like they will get caught.

Reply:
You can add a flag to aid in discovery by the people that bump stage0